test/lit/passes/inlining-optimizing_optimize-level=3.wast

```diff
@@ -3675,25 +3731,6 @@
    (local.get $0)
   )
  )
- ;; CHECK:      (func $___syscall_ret (param $0 i32) (result i32)
```

Comment:
This inlined&removed function happened to match this pattern, in the existing test suite.